### ISSUE TITLE
Add Commontator module to class references

### ISSUE THREAD

**Issue by efmiglioranza** [lml/commontator#97]
I'm migrating an old comment resource in my app that has a Comment class, which conflicts with Commontator classes in the process, for example in:

```
def commontator_set_new_comment(thread, user)
  return if thread.nil? || thread.config.new_comment_style != :t
  new_comment = Comment.new # <== CONFLICT
  new_comment.thread = thread
  new_comment.creator = user
  @new_comment = new_comment if new_comment.can_be_created_by?(user)
end
```

Maybe a proper reference with the module Commontator::Comment can solve this for future cases?

**Comment by Dantemss** [comment]
Yes, that would be desired.

**Comment by Dantemss** [comment]
Will be fixed in 7.0.0 (fixed in 7.0.0.pre.1)
